WoopraSDK/WIdentify.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+
+public class WIdentify {
+    
+    private let wIdnetifyEndPoint = "https://www.woopra.com/track/identify"
+    private let tracker: WTracker
+    
+    init(
+        tracker: WTracker
+    ) {
+        self.tracker = tracker
+    }
+    
+    func run() {
+        let url = URL(string: wIdnetifyEndPoint)!
```

Comment:
I prefer unwrapping the optional value to avoid crashing here.
```guard let url = URL(string: wIdnetifyEndPoint) else {
  // do sth if needed
  return
}
```

Reply:
@michaelrevlis code updated!